Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -142,8 +142,7 @@ class ContentBlockerHelper {
         for name in rules {
             ruleStore.lookUpContentRuleList(forIdentifier: name) { rule, error in
                 guard let rule = rule else {
-                    print("Content blocker load error: " + (error?.localizedDescription ?? "empty rules"))
-                    assert(false)
+                    Sentry.shared.send(message: "Content blocker load \(name) error:  \(error?.localizedDescription ?? "empty rules")")
```

Comment:
Remember to dump the error into `description` 😉  This will help prevent too many messages in the Sentry console.